File: 3194-FindWordsContainingCharacter/3194-FindWordsContainingCharacter.py
# Last updated: 7/31/2025, 5:27:47 PM
class Solution:
    def findWordsContaining(self, words: List[str], x: str) -> List[int]:
        a=list(map(list,words))
        c=[]
        for i in range(len(a)):
            k=a[i].count(x)
            if(k>0):
                c.append(i)
        return c
        

File: 3194-FindWordsContainingCharacter/3194-FindWordsContainingCharacter.java
// Last updated: 7/31/2025, 5:27:43 PM
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> ans = new ArrayList<>();
        for(int i=0; i<words.length; i++){
            if(words[i].indexOf(x)>=0){
                ans.add(i);
            }
        }
        return ans;
    }
}